packages/playground/src/utils/load_deployment.ts

```diff
@@ -27,7 +28,7 @@ export async function loadVms(grid: GridClient, options: LoadVMsOptions = {}) {
         `%c[Error] failed to load deployment with name ${name}:\n${normalizeError(e, "No errors were provided.")}`,
         "color: rgb(207, 102, 121)",
       );
-      return null;
+      failedDeployments.push(name);
```

Comment:
is it the only place we list the machines, and what about the kubernetes list?

Reply:
will add it for kubernetes also

Reply:
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/54944491/337e3b4d-dfc2-410d-bdcc-6a22303645d7)